rootfs/api/models.py

```diff
@@ -1046,7 +1036,7 @@ def save(self, *args, **kwargs):
             config['domains'] = ''
 
         # convert from string to list to work with and filter out empty strings
-        domains = filter(None, config['domains'].split(','))
```

Comment:
`pylint` and other things seem to recommend against the usage of it and `2to3` even converted it when I tried to run that for the lols.

`filter` in 3+ returns an iterator so I had to do a `list()` around it  if I wanted to keep it. Figured might as well go with comprehensions since we anyway use them in other places for similar purposes